CHANGELOG.md

```diff
@@ -45,6 +45,19 @@ All changes to this project will be documented in this file.
 
 ---
 
+## v6.2.4
+
+Release date: **2019-09-20**  
+[Compare code changes][v6.2.4] (🌳 STABLE)
+
+### What’s changed in aragonCLI
+
+## 💡 Feature updates
+
+- Iterate `aragon run`: add `--client-repo` optional argument
+
+---
```

Comment:
This blurb is a placeholder. We need to fill in the correct date & release version before merging.